source/features/shorten-links.tsx

```diff
@@ -5,7 +5,10 @@ import observe from '../helpers/selector-observer.js';
 
 /* This feature is currently so broad that it's not de-inited via signal, it's just run once for all pageloads #5889 */
 function init(): void {
-	observe(`.comment-body a[href]:not(.${linkifiedURLClass})`, shortenLink);
+	observe([
+		`.comment-body a[href]:not(.${linkifiedURLClass})`,
```

Comment:
I'm not sure if we can replace `comment-body` with `markdown-body` here.

Reply:
It seems to me that we were avoiding to apply the feature to markdown comments, but I don't see any explicit decision to do that when blaming.

See live example: https://github.com/refined-github/sandbox/blob/default-a/renamed%20and%20edited.file.md

Let's keep it as is for now